app.rb

```diff
@@ -42,9 +43,35 @@ class App < Sinatra::Base
 
 private
 
+  def handle_notification(payload)
+    ses_notification = JSON.parse(payload['Message'])
```

Comment:
Is this the place that knows about implementation specifics such as SES?

Reply:
Yes, currently all knowledge of SES remains inside of `app.rb`.